README.md

```diff
@@ -44,6 +44,88 @@ docker build -t pca-assessment .
+| start_campaign | The campaign that the assessment should start at. | integer | `1` | no |
+| groups | Consolidated list of email recipients grouped to receive campaigns, [example](#group-dictionary). | list(dictionaries) | | yes |
+| pages | GoPhish landing pages, [example](#page-dictionary). | list(dictionaries) | | yes |
+| campaigns | Assessment campaigns, [example](#campaign-dictionary). | list(dictionaries) | | yes |
+
+### Group Dictionary ###
+
+Groups are imported from a templated `csv` file that can be generated
+with the command `pca-wizard-templates  --targets`.
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| name | Group name in the format of `{assessment identifier}-G{integer}` (e.g. "RV0000-G1"). | string | | yes |
+| targets | List of email recipients, [example](#target-dictionary). | list(dictionaries) | | yes |
+
+### Target Dictionary ###
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| first_name | Recipient's first name. | string | | yes |
+| last_name | Recipient's last name. | string | | yes |
+| email | Recipient's email address. (e.g. "john.doe<span>@target1.tld") | string | | yes |
+| position | Position name for use in creating sub-groups of recipients within the organization such as "HR", "IT", etc. | string | | no |
+
+### Page Dictionary ###
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| name | Page name in the format of `{assessment identifier}-{integer}-{descriptor}` (e.g. "RV0000-1-AutoForward"). | string | | yes |
+| capture_credentials | Capture all non-password input with GoPhish. Supports forwarding after submit action. | boolean | | yes |
+| capture_passwords | Capture password input by the user, but note that captured input is **stored in plain text as of GoPhish version 0.9.0.** | boolean | `False` | no |
+| html | Content of the landing page in HTML format. | string | | yes |
+
+### Campaign Dictionary ###
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| name | Campaign name in the format of `{assessment identifier}-C{integer}` (e.g. "RV0000-C1"). | string | | yes |
+| launch_date | Campaign launch date in 24-hr ISO format with offset. (e.g. "2020-01-01T14:00:00-04:00")  | string | | yes |
+| completed_date | Campaign completion date in 24-hr ISO format with offset. (e.g. "2020-01-01T15:30:00-04:00")  | string | | yes |
+| url | Full URL for the campaign's landing page. (e.g. "http<span>://domain.tld/camp/1") | string | | yes |
+| page_name | Landing page name as defined in the assessment json. | string | | yes |
+| group_name | Group name as defined in the assessment json. | string | | yes |
+| template | Email template for the campaign, [example](#email-template-dictionary). | dictionary | | yes |
```

Comment:
Same thing here (for the type column only):
`dictionary` --> `dict`

Reply:
Changed this in commit 37ea375.